cli/azd/pkg/exec/command_runner.go

```diff
@@ -213,11 +213,15 @@ func redactSensitiveData(msg string) string {
 		"access token": {
 			regexp.MustCompile("\"accessToken\": \".*\""),
 			"\"accessToken\": \"<redacted>\"",
+		},
+		"deployment token": {
+			regexp.MustCompile("--deployment-token .*"),
```

Comment:
Is this greedy capture going to problematic in practice because it will eat the rest of the line (which may have separate arguments)?

Reply:
Yes, I did think that would be the case, but as it was the last one that's passed in it's safe _for now_ 😅

I'm not sure the best way to improve it though, short of figuring out the pattern for the deployment token (I'm assuming that's a standard structure) and regexing against that.